Jenkinsfile_CNP

```diff
@@ -8,8 +8,7 @@ withInfraPipeline(product) {
 
   enableSlackNotifications('#bsp-build-notices')
 
-  after('buildinfra:aat') {
-    currentBuild.result = 'SUCCESS'
-    currentBuild.description = "Stopping deployment to prod temporarily"
+  before('buildinfra:prod') {
+    error('Halting pipeline before PROD.')
```

Comment:
is there exit function instead? would be nicer instead of marking build as failure

Reply:
Not that I am aware of checked the code and it seems that’s the only option other than removing entry from tf approvals.

Reply:
i guess pipeline does same thing when looking for tf approvals